qt/applications/workbench/workbench/config/user.py

```
@@ -155,3 +157,41 @@ def _get_setting(self, option, second=None, type=None):
+    if len(options) < 1:
+        raise RuntimeError("Failed to find QStandardPaths.ConfigLocation")
+    # we only care about those in the home area
+    home_dir = Path.home().expanduser()
+    for candidate in options:
+        # return first one in home directory, likely ~/.config
+        if home_dir in Path(candidate).parents:
+            return Path(candidate)
+    raise RuntimeError("Failed to find QStandardPaths.ConfigLocation in home area")
+
+
+def remove_lock_files(filenames: list[str] = ["mantidproject/mantidworkbench.ini", "QtProject.conf"]) -> None:
```

Ruff (0.16.0)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)